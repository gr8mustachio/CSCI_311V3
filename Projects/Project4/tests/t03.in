10
addEdge 0 9
addEdge 1 2
addEdge 4 1
addEdge 3 2
addEdge 3 4
addEdge 1 5
addEdge 5 6
addEdge 9 6
addEdge 7 6
addEdge 7 8
addEdge 8 9
printGraph
bfs 1
removeEdge 2 1
printGraph
bfs 1
