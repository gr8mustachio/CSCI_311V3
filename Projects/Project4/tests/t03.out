Adjacency lists of the graph are:
9 
2 4 5 
1 3 
2 4 
1 3 
1 6 
5 9 7 
6 8 
7 9 
0 6 8 
BFS on 1
1 2 4 5 3 6 9 7 0 8 
Adjacency lists of the graph are:
9 
5 4 
3 
2 4 
1 3 
1 6 
5 9 7 
6 8 
7 9 
0 6 8 
BFS on 1
1 5 4 6 3 9 7 2 0 8 
